v3/src/pages/docs/core-concepts/activity.md

```diff
@@ -117,6 +113,33 @@ The following is an example of annotating an activity with metadata:
     Description = "A simple activity that writes \"Hello World!\" to the console."
 )]
 ```
+---
+## Outcomes
+
+Custom Activity outcome Nodes can be defined as follows by Annotatting the the Metadata with the "FlowNode" attribute as follows:
```

Comment:
Small typo in "Annotatting" and double occurence of "the" ("the the").
Also, perhaps the capitalization can be changed like this:

"Custom activity outcomes can be defined by annotating the class with the `FlowNodeAttribute` attribute as follows:"